### DISCUSSION TITLE
Spurious (daily?) "Key(s) were updated" notification

### DISCUSSION THREAD

**Discussion by mbwgh** [open-keychain/open-keychain#2835] [Q&A]
I use OpenKeyChain via [Android-Password-Store](https://github.com/android-password-store/Android-Password-Store), installed via F-Droid two weeks ago, version 5.8.2.

I only imported one private key, the one used to decrypt passwords for the aforementioned android [pass](https://www.passwordstore.org/) client.

Every once in a while, I get a "1 key(s) were updated" (or rather the German equivalent) notification.
I am however unable to determine exactly what that means?
Should there even be a reason to apply any updates to manually imported private keys?
I *believe* I get this notification once a day, no matter whether I have used the key on that day or not.

For what it's worth, I get a "Network error (SSLProtocolException) for 'hkps://keys.openpgp.org'. Check your Internet connection!" whenever I click on "Update all keys". Might this be related to OpenKeychain trying (and failing) to update all keys once a day?

**Comment by 532910** [comment]
1. In fact you imported a keypair, not just a private key. So it tries to update your public key.

2. You can disable keys update in `Settings -> Synchronization`.

3. Does OpenKeyChain have network access? (`Android settings -> Apps -> OpenKeyChain -> Mobile data and Wi-Fi`)

4. Do you have the same SSL errors for https://keys.openpgp.org in browser?

**Reply by mbwgh** [reply]
Hello sergio,

thanks for clearing up what the update notifications are about.

As far as I'm concerned, OpenKeyChain should have network access. I have a fairly old mobile phone using Android 7.0, maybe that has something to do with it (expired certificate?).

Going in the settings, I only saw "Allow background data usage" allowing it to use data if data saving mode is active, both of which I allowed.
Otherwise, I only seem to be able to allow camera and memory access.

I can open keys.openpgp.org both in Firefox and Chrome and can find my public key with the correct fingerprint there.

In any case, why should I update the public key if I have the private one (and can derive it) and it hasn't even expired?
Is this about checking for revocation?
